homeassistant/components/samsungtv/bridge.py

```diff
@@ -705,9 +705,8 @@ async def async_device_info(self) -> dict[str, Any] | None:
             LOGGER.debug("Device info on %s is: %s", self.host, device_info)
```

Comment:
Should this block be indented one more level?

Reply:
I think you're right. That's a bug from a previous PR.

Reply:
I also spotted an issue with the port so I've fixed that